### ISSUE TITLE
 empty cor.fa files

### ISSUE THREAD

**Issue by jazberna1** [jgurtowski/ectools#7]
Hello James,

I believe I have successfully run ectools as I don't see errors in the logs. 

However I see most of the cor.fa files are empty, not their corresponding delta and pileup files.
Does that mean the read correction was either not necessary/possible ?

Thanks
Jorge

**Comment by jgurtowski** [comment]
Yes, it is possible. What percentage of the input reads were successfully
corrected? Another issue could be the length of the input reads, if they
are too short, they will likely not be corrected.

**Comment by jazberna1** [comment]
Hello James,

I will send you the proportion input reads were successfully corrected asap, it's just I decided to rerun to double check. It's still running, but it was low.

I am using pacbio reads longer than 1kb that I initially got from here:
http://labshare.cshl.edu/shares/schatzlab/www-data/ectools/w303/Pacbio.fasta.gz

Am I correct saying that once the correction is done I have to create a final dataset where the corrected reads replace the originals (and leave the uncorrected reads as the are)

Thanks
Jorge

**Comment by jgurtowski** [comment]
Ok, and are you using the associated Illumina reads from W303 to do the
correction?

I'm not sure I understand your last sentence, but you want to use the
corrected reads that are output. The uncorrected reads are not very useful
for assembly.

All of this being said, depending on what you are trying to accomplish,
there are some more modern assemblers available today that do correction
for you: https://github.com/marbl/canu

**Comment by jazberna1** [comment]
Hello James,

Yes, I used the following files to create the contigs that then used to correct the pacbio reads:
http://labshare.cshl.edu/shares/schatzlab/www-data/ectools/w303/Illumina_500bp_2x300_R1.fastq.gz
http://labshare.cshl.edu/shares/schatzlab/www-data/ectools/w303/Illumina_500bp_2x300_R2.fastq.gz

Ok, I see only corrected reads are good for assembly.

I am thinking that I might have done something wrong while creating the contigs and that had impact on the subsequent correction.

I'll have a look to  https://github.com/marbl/canu

Thanks,
Jorge
